neuromation/api/plugins.py

```diff
@@ -0,0 +1,41 @@
+import numbers
+from typing import Any, Dict, Type, Union
+
+
+class ConfigConflict(Exception):
+    pass
+
+
+class PluginManager:
+
+    _config_spec: Dict[str, Any]
+
+    def __init__(self, config_spec: Dict[str, Dict[str, Any]]):
+        self._config_spec = config_spec
+
+    def _register_config_param(
+        self,
+        section: str,
+        name: str,
+        type: Union[Type[bool], Type[numbers.Real], Type[numbers.Integral], Type[str]],
+    ) -> None:
+        if section == "alias":
+            raise ConfigConflict("Registering aliases is not supported yet.")
+        if section in self._config_spec and name in self._config_spec[section]:
+            raise ConfigConflict(
+                f"Config parameter {section}.{name} already registered"
+            )
+        self._config_spec.setdefault(section, dict())
+        self._config_spec[section][name] = type
+
+    def register_config_int(self, section: str, name: str) -> None:
```

Comment:
I know, I had proposed this name but it looks a little awkward.  Maybe you can suggest better naming?